docs/plat/meson-gxl.rst

```diff
@@ -0,0 +1,27 @@
+Trusted Firmware-A for Amlogic Meson S905x (GXL)
```

Comment:
Update the list of paths in ``maintainers.rst`` and add this platform to the list of supported platforms in the readme.